static/backbone/js/api/mapplication.js

```diff
@@ -39,16 +47,24 @@ define(["marionette",
         });
 
         Mapplication.addInitializer(function (options) {
-            options.projects = new Projects();
+            this.availableProjects = new Projects();
+            this.selectedLayers = new Layers();
```

Comment:
Removed "availableProjects" and "selectedLayers" from global object. These are part of opts again.